.drone.yml

```diff
@@ -25,7 +25,7 @@ pipeline:
           exit 1 ;
         fi
       - if [ -n "$DRONE_PULL_REQUEST" ] ; then cd testing/cache && sbt ++$SCALA_VERSION ensimeConfig ensimeServerIndex && cd ../.. ; fi
-      - if [ -n "$DRONE_PULL_REQUEST" ] && [ "$SCALA_VERSION" = "2.12.2" ] ; then sbt ++$SCALA_VERSION "core/it:testOnly *BasicWorkflow*" ; fi
+      - if [ -n "$DRONE_PULL_REQUEST" ] ; then sbt ++$SCALA_VERSION "core/it:testOnly *BasicWorkflow*" ; fi
```

Comment:
to control the version of scala, remove entries from the matrix at the bottom of the file.